node/__tests__/getExternalData.test.ts

```diff
@@ -0,0 +1,44 @@
+// this is because of jest not recognizing type imports
+/* eslint-disable @typescript-eslint/consistent-type-imports */
+import TikTokExternalData from '../clients/tikTokExternalData'
+import { EXTERNAL_DATA_MOCK } from '../mocks/externalDataMock'
+import { externalDataService } from '../services/externalDataService'
+import { ExternalData } from '../typings/externalData'
+
+describe('getExternalData', () => {
+  it('should parse the API response to an ExternalData object', async () => {
+    const client = externalDataService({
+      getTikTokData: () => EXTERNAL_DATA_MOCK,
+    } as unknown as TikTokExternalData)
+
+    const externalData = await client.getExternalData()
+
+    const expectedResult = {
+      countries: [
+        {
+          code: 'CA',
+          name: 'Canada',
+          currencies: ['CAD', 'USD'],
+        },
+        {
+          code: 'FR',
+          name: 'France',
+          currencies: ['EUR', 'USD', 'GBP'],
+        },
+      ],
+      timezones: [
+        { name: 'America/Sao_Paulo', value: 'GMT-03:00' },
+        { name: 'Pacific/Honolulu', value: 'GMT-10:00' },
+      ],
+      industries: [
+        {
+          name: 'Sports & Entertainment',
+          children: [{ id: '292104', name: 'Musical Instruments' }],
+        },
+      ],
+      phoneAreas: [{ countryCode: 'AD', areaCode: '+376' }],
```

Comment:
```suggestion
      phoneAreas: [{ countryCode: 'AD', name: 'Andorra', areaCode: '+376' }],
```
Is the country name not necessary for new cases we still don't have on the map for internationalization?

Reply:
You're right! I'm going to include the name (both for `phoneAreas` and `countries`)